ts/Core/Axis/Axis3DComposition.ts

```diff
@@ -69,12 +70,14 @@ declare module './AxisOptions' {
     }
 }
```

Comment:
Shouldn't these options have doclets instead of the ones in Axis3DOptions' `Axis3DTitleOptions` and `Axis3DTitleOptions` interfaces? There's the upcoming "make options optional" task, so if this change should be done there a note with the key todo points should suffice IMO.